api/swagger.yml

```diff
@@ -603,12 +603,45 @@ components:
           type: boolean
       required:
         - id
+
+    LoginConfig:
+      type: object
+      properties:
+        login_url:
+          description: primary URL to use for login.
+          type: string
+        login_failed_message:
+          description: |
+            message to display to users who fail to login; a full sentence that is rendered
+            in HTML and may contain a link to a secondary login method
+          type: string
+        fallback_login_url:
+          description: secondary URL to offer users to use for login.
+          type: string
+        fallback_login_label:
+          description: label to place on fallback_login_url.
+          type: string
+        login_cookies:
+          description: cookies to store JWT
```

Comment:
```suggestion
          description: cookie names used to store JWT
```

Reply:
Possibly also change the property name to `login_cookie_names`. Not feeling too strongly about this one.